docs/pages/includes/database-access/database-config.yaml

```diff
@@ -53,14 +54,41 @@ db_service:
       # When this option is set the Database Agent doesn't try to check the MySQL server version.
       server_version: 8.0.28
 
-    # AWS specific configuration, only required for RDS/Aurora/Redshift.
+    # Optional AWS configuration for AWS hosted databases. AWS region and
+    # service specific configurations can usually be auto-detected from the
+    # endpoint.
     aws:
       # Region the database is deployed in.
       region: "us-east-1"
       # Redshift specific configuration.
       redshift:
         # Redshift cluster identifier.
         cluster_id: "redshift-cluster-1"
+      # RDS specific configuration.
```

Comment:
Since "RDS-specific" is a compound modifier that comes before the noun, we should hyphenate it. Same for the other similar comments, e.g., "ElastiCache-specific" and "Redshift-specific".